lang/en/texts/android.html

```diff
@@ -0,0 +1,30 @@
+<h4>Mobile</h4>
+<a download="4. Additives - Mobile - EN.png" href="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/4. Additives - Mobile - EN.png" target="_blank"><img width="300px" src="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/4. Additives - Mobile - EN.png"></a>
```

Comment:
The `href` and `src` should be URL-encoded: `https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/4.%20Additives%20-%20Mobile%20-%20EN.png` (fails basic HTML validation as-is)